stubs/regex/regex/_regex_core.pyi

```diff
@@ -0,0 +1,45 @@
+from typing import Any, AnyStr, Callable
+
+class error(Exception):
+    def __init__(self, message: str, pattern: AnyStr | None = ..., pos: int | None = ...) -> None: ...
+
+A: int
+ASCII: int
+B: int
+BESTMATCH: int
+D: int
+DEBUG: int
+E: int
+ENHANCEMATCH: int
+F: int
+FULLCASE: int
+I: int
+IGNORECASE: int
+L: int
+LOCALE: int
+M: int
+MULTILINE: int
+P: int
+POSIX: int
+R: int
+REVERSE: int
+T: int
+TEMPLATE: int
+S: int
+DOTALL: int
+U: int
+UNICODE: int
+V0: int
+VERSION0: int
+V1: int
+VERSION1: int
+W: int
+WORD: int
+X: int
+VERBOSE: int
+
+DEFAULT_VERSION: int
+
+class Scanner:
+    def __init__(self, lexicon: list[tuple[AnyStr, Callable[[Scanner, AnyStr], Any] | Any | None]], flags: int = ...) -> None: ...
+    def scan(self, string: AnyStr) -> tuple[list[Any], AnyStr]: ...
```

Comment:
This class is included in `__all__` in `regex/__init__.py` but is undocumented. The only reference I could find to it was this unit test: https://github.com/mrabarnett/mrab-regex/blob/1c30637ec7568168decb853c9cb0f889ed524b29/regex_3/test_regex.py#L805

Reply:
The standard library has the same situation, and we decided to leave it out: #6405

I guess `regex.Scanner` exists just for compatibility with the stdlib `re` module, so we should leave it out too.

Reply:
Cool I've removed the class 😄